pipenv/resolver.py

```diff
@@ -12,7 +12,7 @@
 
 def find_site_path(pkg, site_dir=None):
     import pkg_resources
-    if site_dir is not None:
+    if site_dir is None:
```

Comment:
Eh?

Reply:
In fact the `site_dir` is always given the same value as `os.path.dirname(os.path.dirname(os.path.abspath(__file__)))`, lol